components/exhibition/ExhibitionChaptersNavigation.vue

```diff
@@ -0,0 +1,90 @@
+<template>
+  <div class="my-3 d-flex justify-content-end">
+    <b-link
+      v-if="previousChapter"
+      :to="previousChapter.url"
+      class="chapter-nav prev"
+      data-qa="previous chapter button"
+    >
+      {{ previousChapter.name }}
+    </b-link>
+    <b-link
+      v-if="nextChapter"
+      :to="nextChapter.url"
+      class="chapter-nav next"
+      data-qa="next chapter button"
+    >
+      {{ nextChapter.name }}
+    </b-link>
+  </div>
+</template>
+
+<script>
+  export default {
+    props: {
+      exhibitionIdentifier: {
+        type: String,
+        required: true
+      },
+      chapters: {
```

Comment:
As this component is concerned only with navigation, this property should already be an array of objects having only `name` and `url` properties, assembled by the parent component.